core/src/main/java/com/github/kaktushose/jda/commands/dispatching/Runtime.java

```
@@ -107,7 +108,7 @@ private void executeHandler(GenericInteractionCreateEvent incomingEvent) {
             case GenericComponentInteractionCreateEvent event -> componentHandler.accept(event, this);
             case ModalInteractionEvent event -> modalHandler.accept(event, this);
             default ->
-                    throw new IllegalStateException("Should not occur. Please report this error to the devs of jda-commands.");
+                    throw new JDACException.Internal("Should not occur. Please report this error to the devs of jda-commands.");
```

"Please report this error to the devs of jda-commands." can be removed, would be duplicated